libbeat/tests/system/test_dashboard.py

```diff
@@ -183,12 +184,13 @@ def test_export_dashboard_cmd_export_dashboard_by_id_unknown_id(self):
                         "-E", "setup.kibana.protocol=http",
                         "-E", "setup.kibana.host=" + self.get_kibana_host(),
                         "-E", "setup.kibana.port=" + self.get_kibana_port(),
-                        "-id", "No-such-dashboard"]
+                        "-id", dashboard_id]
         )
 
         beat.check_wait(exit_code=1)
 
-        assert self.log_contains("error exporting dashboard: Not found") is True
+        expected_error = "error exporting dashboard: Saved object [dashboard/{}] not found".format(dashboard_id)
```

Comment:
We should probably make it pass on both older and newer versions. Like when running TEST_ENVIRONMENT=latest.

Reply:
Thanks for the suggestion! Added regexp as in https://github.com/elastic/beats/pull/17773/commits/f6c56fe445eb6ec442e75c2bbb92ec9a2865b408.